README.md

```diff
@@ -1,5 +1,15 @@
 # interview-eng-i
 
+## Introduction
+Project Tiro is a simple web application built to help developer candidates demonstrate practical skills with
```

Comment:
@agbasioscar This is an internal project name, I don't think we need to reference it in the candidate-facing docs.